src/main/kotlin/kweb/Kweb.kt

```diff
@@ -298,6 +300,7 @@ class Kweb private constructor(
                     }
                 } catch (e: Exception) {
                     logger.error("Exception while receiving websocket message", e)
+                    kwebConfig.onWebsocketMessageHandlingFailure(e)
```

Comment:
Just curious, was this swallowed in your code because you didn't have a logger or couldn't see the logs?